connectors/iotf/src/main/java/quarks/connectors/iotf/runtime/IotfDeviceCommands.java

```diff
@@ -41,8 +45,7 @@ public void accept(Consumer<Command> commandSubmitter) {
         try {
             connector.subscribeCommands(commandSubmitter);
         } catch (Exception e) {
-            // TODO Auto-generated catch block
-            e.printStackTrace();
+			logger.error("Exception caught while subscribing commands: {}", e);
```

Comment:
Thank you for the review @dlaboss. I will remove placeholder "{}" from my changes.